### DISCUSSION TITLE
Is this a bug or a new design? version:7.6.0

### DISCUSSION THREAD

**Discussion by Tomycl** [mantinedev/mantine#5828] [Q&A] [answered]
from `7.5.4` to `7.6.0`，<Tab/> `ouline` error: we lost the lines..
<img width="804" alt="image" src="https://github.com/mantinedev/mantine/assets/33180830/df8209f4-fbd8-4d7c-8a3a-ea410ec6d63e">

**Comment by rtivital** [comment, marked as answer]
It will be fixed in the next patch

**Reply by Tomycl** [reply]
Many thanks!
